schema/playground/cube.schema

```diff
@@ -0,0 +1,7 @@
+package playground;
+import "improbable/vector3.schema";
+
+component Cube {
+    id = 12008;
+    improbable.Vector3f velocity = 1;
```

Comment:
It isn't immediately obvious what this is. Calling it target_velocity would be a little better.

Generally a component should be the definition of some behaviour and the data definition should give you most of what you need to know about it . So having a `CubeTargetVelocity` component would probably make more sense. We could also add this to other entity types when we increase the scope of the playground.